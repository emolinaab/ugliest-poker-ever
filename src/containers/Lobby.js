import React from 'react';

const Lobby = () => (
  <section></section>
);

export default Lobby;
