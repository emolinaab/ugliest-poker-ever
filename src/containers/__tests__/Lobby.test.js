import React from 'react';
import { shallow } from 'enzyme';
import Lobby from '../Lobby';

describe('Lobby Component', () => {
  it('should render correctly', () => {
    const component = shallow(<Lobby />);
  
    expect(component).toMatchSnapshot();
  });
});
