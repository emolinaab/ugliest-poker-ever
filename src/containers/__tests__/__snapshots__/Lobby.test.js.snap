// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Lobby Component should render correctly 1`] = `<section />`;
